website/docs/bottomsheet.md

```diff
@@ -42,9 +42,13 @@ const list = [
 
 ## Props
 
-- [`containerStyle`](#containerStyle)
-- [`isVisible`](#isvisible)
-- [`modalProps`](#modalprops)
+- [Usage](#usage)
```

Comment:
Not sure what happened here. These lines should not be added here.

Reply:
Not sure what happened with my editor. It is automatically generated. And it has been reverted.